go.mod

```diff
@@ -1,6 +1,6 @@
 module go.kubeguard.dev/guard
 
-go 1.17
+go 1.18
```

Comment:
should this change be separate from this PR?

Reply:
We are already using 1.18 because of https://github.com/kubeguard/guard/commit/61066e661e123171810bd061060eb6c54669ebfd pr. Go mod wasn't updated i guess there